src/Sylius/Bundle/CoreBundle/Console/Command/SetupCommand.php

```diff
@@ -180,6 +182,26 @@ private function getAdministratorUsername(InputInterface $input, OutputInterface
         return $username;
     }
 
+    private function getAdministratorFirstName(InputInterface $input, OutputInterface $output): string
+    {
+        /** @var QuestionHelper $questionHelper */
+        $questionHelper = $this->getHelper('question');
+        $question = new Question('Please enter your first name (press enter to leave blank): ', '');
```

Comment:
For the sake of consistency with the other questions, I'd suggest the following change:

```suggestion
        $question = new Question('First name (press enter to leave blank): ', '');
```

<img width="542" height="418" alt="Image" src="https://github.com/user-attachments/assets/86e84418-7f15-42b0-9d5b-b7bfe3695d20" />